w3/domain/usecase/user_it_usecase.go

```diff
@@ -58,7 +59,10 @@ func (u *userITUsecase) RegisterUserIT(userITRequest UserITRegisterRequest) (Use
 	}
 	user.Name = userITRequest.Name
 	user.NIP = userNip
-	user.Password = hashedPassword
+	user.Password = sql.NullString{
```

Comment:
ini juga

Reply:
It's not about go-validator, but as we know that password is nullable (ERD)